csrankings-a.csv

```diff
@@ -1436,6 +1436,7 @@ Andrew J. Davison,Imperial College London,https://www.doc.ic.ac.uk/~ajd,SNQFnBEA
 Andrew J. McAllister,University of New Brunswick,http://www.cs.unb.ca/~andrewm,d1yVDNgAAAAJ
 Andrew J. Parkes,University of Nottingham,https://www.nottingham.ac.uk/computerscience/people/andrew.parkes,G0yAJAwAAAAJ
 Andrew J. Wellings,University of York,https://www-users.cs.york.ac.uk/~andy,yPa4QrkAAAAJ
+Andrew K. Hirsch,University at Buffalo,https://cse.buffalo.edu/~akhirsch,P_rIn1sAAAAJ

```

Comment:
No such web page.

Reply:
For now, this will suffice: https://engineering.buffalo.edu/computer-science-engineering/people/faculty-directory/full-time.host.html/content/shared/engineering/computer-science-engineering/profiles/faculty/ladder/hirsch-andrew.detail.html

Reply:
Update. Thank you!